<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,4,20,3,1">
    <item row="0" column="3">
     <widget class="QWidget" name="widget_board" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widget_board{
	background: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0.45 rgba(4, 51, 255, 110), stop:0.55 rgba(255, 38, 0, 110));
	border-radius: 30px;
}
QLabel#label_blueteam_name
{
    border: 8px solid blue;
	background: rgba(0, 150, 255, 97);
    padding: 15px;
    border-radius: 3px;
	font: bold 20px;
}

QLabel#label_redteam_name
{
    border: 8px solid red;
	background: rgba(255, 38, 0, 97);
    padding: 15px;
    border-radius: 3px;
	font: bold 20px;
}

QLabel#label_blueteam_dir
{
    border: 5px solid blue;
	border-right: none;
    padding: 7px;
    border-radius: 0px;
	font: bold 13px;
}
QLabel#label_blueteam_freq
{
    border: 5px solid blue;
	border-left: none;
    padding: 7px;
    border-radius: 0px;
	font: bold 13px;
}

QLabel#label_redteam_dir
{
    border: 5px solid red;
	border-right: none;
    padding: 7px;
    border-radius: 0px;
	font: bold 13px;
}
QLabel#label_redteam_freq
{
    border: 5px solid red;
	border-left: none;
    padding: 7px;
    border-radius: 0px;
	font: bold 13px;
}

QLabel#label_blueteam_score
{
    border: 7px solid blue;
	background: rgba(0, 150, 255, 97);
    padding: 7px;
    border-radius: 16px;
	font: bold 28px;
}
QLabel#label_redteam_score
{
    border: 7px solid red;
	background: rgba(255, 38, 0, 97);
    padding: 7px;
    border-radius: 16px;
	font: bold 28px;
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,2,1,1,5,3,1,3,5,1,1,2,1">
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_blueteam_score">
         <property name="text">
          <string>Score</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>89</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_redteam_dir">
         <property name="text">
          <string>Red Direction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_blueteam_freq">
         <property name="text">
          <string>Blue Freq</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>89</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLabel" name="label_redteam_name">
         <property name="text">
          <string>Red Team Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_redteam_score">
         <property name="text">
          <string>Score</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_blueteam_dir">
         <property name="text">
          <string>Blue Direction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_blueteam_name">
         <property name="accessibleName">
          <string/>
         </property>
         <property name="text">
          <string>Blue Team Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_redteam_freq">
         <property name="text">
          <string>Red Freq</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_game" native="true"/>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>724</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_config" native="true">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid black;
	border-radius: 5px;
    margin-top: 1em;
}

QGroupBox::title {
	border: 2px solid black;
	border-radius: 5px;
	color: green;
    subcontrol-origin: margin;
    left: 10;
    padding: 1 10 1 10;
}

QLabel {
    border: 2px dashed black;
    padding: 3px;
    border-radius: 3px;
	font: bold 13px;
}

QLineEdit {
    border: 2px dashed black;
    border-radius: 3px;
    padding: 2 8px;
    background: rgb(214, 214, 214);
    selection-background-color: darkgray;
}

QPushButton {
    background-color: rgb(107, 255, 10);
    border: 2px solid black;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}
QPushButton:hover {
    background-color: rgb(15, 172, 0);
    border: 2px solid black;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}
QPushButton:pressed {
    background-color: rgb(15, 172, 0);
    border: 3px solid black;
    border-radius: 10px;
    font: bold 13px;
    padding: 4px;
}

QCheckBox {
    spacing: 10px;
    border: 2px dashed black;
	padding: 0 3 0 3;

}

QCheckBox::indicator {
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6" rowstretch="20,20,10,30,30,5,10,10,10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Command</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_command_ip"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>IP</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Blue Team</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_blueteam_name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_blueteam_port"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_blueteam_handy">
            <property name="text">
             <string>control with arrow keys</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pushButton_resetgame">
         <property name="text">
          <string>Reset Game</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_resetround">
         <property name="text">
          <string>Reset Round</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Red Team</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_redteam_port"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_redteam_name"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_redteam_handy">
            <property name="text">
             <string>control with w,s,a,d</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Field</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_field_cellnum"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_field_cellwidth"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Cell width</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cell num</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_start_stop">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Broadcast</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>IP</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_broadcast_ip"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_broadcast_port"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>724</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
